pkgs/development/python-modules/coinmetrics-api-client/default.nix

```
@@ -0,0 +1,42 @@
+{ buildPythonPackage, fetchPypi, lib, orjson, pandas, poetry-core
+, pytestCheckHook, pytest-mock, pythonOlder, python-dateutil, requests, typer
+, websocket-client, fetchFromGitHub }:
```

```suggestion
, websocket-client }:
```
Unused inputs should be removed. I did not double check if there are any other unused ones.

---

Good catch. I am going to mark this as a draft for now.